mesonbuild/optinterpreter.py

```diff
@@ -77,9 +77,6 @@ def __init__(self, subproject, command_line_options):
             if self.subproject != '': # Strip the beginning.
                 if not o.startswith(self.sbprefix):
                     continue
-            else:
-                if ':' in o:
-                    continue
```

Comment:
Sorry, dunno what sort brain aneyrysm I was having there. Of course this is about values, not names. 🤦‍♂️